src/popup/App.vue

```diff
@@ -64,11 +64,11 @@
 
             <md-table-row v-for="log in displayLogs" :key="log.id">
               <md-table-cell class="no-wrap">
-                <md-checkbox v-if=" log.issue === 'NO ID' || log.isSynced || log.duration < 60" v-model="checkedLogs" disabled :value="log" />
+                <md-checkbox v-if="!logCanSync(log) || isRefreshingEntries || isSyncing" v-model="checkedLogs" disabled :value="log" />
                 <md-checkbox v-else v-model="checkedLogs" :value="log" />
```

Comment:
Could the two versions of the line here with if-else be replaced by one version with :disabled="!logCanSync(log) || isRefreshingEntries || isSyncing" maybe?